public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -149,25 +154,34 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
             <ToolbarButton
               variant="canvas"
               key="split"
-              tooltip="Split the pane"
+              tooltip={t('explore.toolbar.split-tooltip', 'Split the pane')}
               onClick={onOpenSplitView}
               icon="columns"
               disabled={isLive}
             >
-              Split
+              <Trans i18nKey="explore.toolbar.split-title"> Split </Trans>
```

Comment:
```suggestion
              <Trans i18nKey="explore.toolbar.split-title">Split</Trans>
```

Spaces around "Split" seem to be redundant. I don't think they are needed / rendered. Translation files seem to trim them anyway 🤷

Reply:
💯 Sure, I have removed the spaces.